rakelib/plugins_docs_dependencies.rake

```diff
@@ -204,14 +203,6 @@ task :generate_plugins_version do
         end
       end
     end
-    DepProxy.class_eval do
```

Comment:
@mashhurs I think it is ok. As per @jsvd description on the PR:

`since https://github.com/rubygems/rubygems/pull/5698 Bundler no longer has a DepProxy class, so we can remove the tweaks for it.`

Then it sounds like this no longer exists, and not needed.